experiments/pages/docusign/autoPopulationTutorial.md

```diff
@@ -0,0 +1,135 @@
+
+
+## DocuSign Auto-Populate Integration Set-up
+ 
+##### What you’ll need: 
+1.	A DocuSign sandbox account (create one here: https://go.docusign.com/trial/).
+2.	A Skuid page with data you want to push to a DocuSign template.
+3.	(Eventually) admin access to the production DocuSign account that will be sending the template document for signatures.
+
+##### DocuSign : 
+1.	In your sandbox account, navigate to the admin section. 
+2.	From the menu on the left, select “API and Keys”.
+3.	Click “Add Integrator Key”, define a description and save
+4.	Select your new integrator key and create the correct Skuid Redirect URIs (generate yours with this tutorial: https://docs.skuid.com/latest/en/data/callback-urls-redirect-uris.html)
+5.	Generate a secret key and store it somewhere safe. (Warning: you will only be able to access it one time). Click Save. 
+6.	Go back to DocuSign 
+7.	Create a template and upload the document that is to be populated and sent 
+8.	Define recipients roles and signing order. (If the recipient is going to be variable, only fill in the role name)
+9.	Create signature tabs and define a role for each tab. 
+10.	Create plain text tabs (fields) for each data item that you want to populate.(Make sure to specify the recipient). 
+a.	Define a custom data label (no spaces) for each tab. 
+11.	Take note of the Template Id (the Id can be found in the URL when viewing the document)
+ 
+
+##### Skuid Configure: 
+1.	Go to Data Sources > Authentication Providers
+2.	Create a new Authentication Provider with a unique name and DocuSign Provider type. 
+3.	Set the Authorize Endpoint URL and Token Endpoint URL to:
+a.	 Authorize: https://account-d.docusign.com/oauth/auth
+b.	Token: https://account-d.docusign.com/oauth/token
+4.	Set the Client Id to the Integrator Key Id defined in DocuSign
+5.	Set the client secret to the Secret Key you saved from DocuSign earlier. Save! 
+6.	Now go to Data Sources > Data Sources
+7.	Create a New Data Source with a unique name and with the REST Data Source Type
+8.	Set the URL to  https://demo.docusign.net/restapi and the Authentication Provider to DocuSign. Save. 
+
+
+##### Skuid Page:
+1.	Create a REST api model on the DocuSign data source with Read/Write Model Behavior.
+2.	On this new model, select Methods>insert
+3.	Set values as follows: 
+a.	type =Insert
+b.	url = /accounts/{{AccountId}}/envelopes/
+c.	HTTP verb =POST 
+d.	Send new field values = “As templated request body” 
```

Comment:
Wait, I'm confused --- are you using a custom payload request snippet? Or are you using a templated request body? Which are you doing? Ideally --- you'd use the templated request body, so you don't have to write a snippet.

Reply:
I am using a templated request body, I think I just got the terms confused. I'll update so that it's consistent throughout the document.